native/src/routes/landing/components/CitySelector.js

```diff
@@ -54,6 +54,8 @@ class CitySelector extends React.PureComponent<PropsType> {
     const cities = this.props.cities
     if (normalizedFilter === 'wirschaffendas') {
       return cities.filter(_city => !_city.live)
+    } else if (__DEV__) {
```

Comment:
Use there the flag from the build config.

Reply:
It proved useful to me to display all cities when developing. If you want I can also remove this again.

Reply:
I like it, leave it :)

Reply:
Can stay. The flag from the build config should be used.